test/functional/test_site/testTriggerAttributeHonor.md

```diff
@@ -0,0 +1,8 @@
+**Popover initiated by trigger should honor trigger attribute**
```

Comment:
This is definitely just a nit, but `testTriggerAttributeHonor.md` sounds a little odd to me. What do you think of `testTriggerAttributeHonor.md` -> `testPopoverTrigger.md`?

Reply:
Ya, agreed. Makes sense